src/core/Tooltip/TooltipContent/TooltipContent.baseStyles.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+
+export const baseStyles = (arrowOffsetPx: number, theme: SuomifiTheme) => css`
+  margin-top: 10px;
+  margin-bottom: 10px;
+  ${theme.typography.bodyTextSmall};
+  position: relative;
+  border: 1px solid ${theme.colors.depthDark2};
+  border-radius: ${theme.radius.basic};
+  background-color: ${theme.colors.highlightLight4};
+  padding: 20px 43px 20px 20px;
```

Comment:
Should we use inset spacing tokens?

Reply:
True, maybe we could use `insetXxl` token here.